wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -0,0 +1,12 @@
+# -*- encoding : utf-8 -*-
+
+require 'decko/engine'
+
+Rails.application.routes.draw do
+
+  if !Rails.env.production? && Object.const_defined?( :JasmineRails )
+    mount Object.const_get(:JasmineRails).const_get(:Engine) => "<%= mount_point %>/specs"
+  end
```

Comment:
perhaps if you're doing core-dev, yeah.  But a normal decko-rails user should definitely not have this stuff.